packages/api/src/tx/msg.ts

```diff
@@ -37,9 +36,6 @@ export async function setupTxExtension(
   messageTypeRegistry.forEach((value, key) => {
     customRegistry.push([`/${key}`, value]);
   });
-  regenRegistry.forEach(([key, value]) => {
```

Comment:
deleting this. As @blushi pointed out, the `messageTypeRegistry` already registers all the custom regen msgs.